globus_sdk/auth/token_response.py

```diff
@@ -56,11 +60,19 @@ def by_resource_server(self):
         return self._by_resource_server
 
     @property
+    def expires_at(self):
+        """
+        A POSIX timestamp representation of the expiration
+        time for the top-level token in the response.
+        """
+        return self['expires_at']
+
+    @property
     def expires_in(self):
         """
         The ``expires_in`` value for the top-level token in the response.
         """
-        return self['expires_in']
+        return int(self['expires_at'] - time.time())
```

Comment:
As with `self.expires_at`, I think this is problematic. Maybe drop this one too? Maybe drop all of the top-level token properties?